pypeit/spectrographs/keck_lris.py

```diff
@@ -1295,7 +1397,7 @@ def configuration_keys(self):
             and used to constuct the :class:`~pypeit.metadata.PypeItMetaData`
             object.
         """
-        return super().configuration_keys() + ['dispangle', 'cenwave', 'amp', 'binning']
+        return super().configuration_keys() + ['dispangle', 'cenwave']
```

Comment:
please remind me why we don't need `amp` or `binning` here

Reply:
I removed them here (for LRIS RED) because they are already added in the parent class:
https://github.com/pypeit/PypeIt/pull/1666/files#diff-e7e6fa411dc194e8506c4be0d327eba64e5808bcaabd921f2abee3bb415a9e8dR261